app/Http/Controllers/ArticleController.php

```diff
@@ -70,17 +71,17 @@ public function details()
         }
     }
 
-    //return array with the count of each status
-
+    /**
+     * @return JsonResponse
+     */
     public function overview()
     {
-        $Articles = [];
-        $Articles['total'] = Article::count();
-        $Articles['star'] = Article::where('star_ind', '1')->count();
-        $Articles['read'] = Article::where('status', 'read')->count();
-        $Articles['unread'] = Article::where('status', 'unread')->count();
+        $total  = Article::count();
+        $star   = Article::where('star_ind', '1')->count();
+        $read   = Article::where('status', 'read')->count();
+        $unread = $total - $read;
 
-        return response()->json($Articles);
+        return new JsonResponse(compact('total', 'star', 'read', 'unread'));
```

Comment:
Always good to get the query count down so a sensible change, but why are you using new JsonResponse instead of the response->json() method? It returns "Symfony\Component\HttpFoundation\Response" ( http://laravel.com/api/5.0/Illuminate/Contracts/Routing/ResponseFactory.html#method_json) seems a bit cleaner in my book but please let me know if I've missed something.